bot/siteMonitor.py

```diff
@@ -0,0 +1,98 @@
+import urllib.request
+import difflib
+import time
+from bs4 import BeautifulSoup
+import re
+
+request_time = 10
+    
+    
+    
+#code for diff
+def diff(first, second):
+        
+        item_list = []
+        
+        second = set(second)
+        
+        for item in first:
+            if item not in second:
+                item_list.append(item)
+        
+        return item_list
+
+
+
+def takeDifferences(lists, i):
+    
+    
+    for new, old in zip(lists[i+1], lists[i]):
+        
+        differences = diff(new, old)
+        
+        #not empty list
+        if(differences != []):
+            if len(differences) < 2:
+                print(differences)
+            else:
+                print("Lot of news pal!")    
+        else:
+           # print("We have to go deeper")
+            if i < len(lists)-3:
+                i += 2
+                takeDifferences(lists, i)
+            else:
+                break    
+            #save previous changes
+
+    
+#save site content    
+url = "https://www.wykop.pl/wykopalisko/najnowsze/"
+html = urllib.request.urlopen(url).read()
+soup = BeautifulSoup(html, "html.parser")
+
+#repeat
+while(True):
+    
+    time.sleep(request_time)
+    url2 = "https://www.wykop.pl/wykopalisko/najnowsze/"
```

Comment:
I would try writing this code as a function, starting from this line, with url as a parameter. We need a more general approach, then put that wykop.pl site as an example in other file by importing the module.